Waterminder/Waterminder WatchKit Extension/Screens/WaterViewModel.swift

```diff
@@ -6,6 +6,16 @@ class WaterViewModel: ObservableObject {
     var drinkingAmount: Double = 100.0
     var drinkingTarget = 2000.0
     var waterLevel: CGFloat = .zero
+    var firstLaunchKey = "firstLaunch"
```

Comment:
This key should be a let constant. And maybe this name is not the best to describe the first interaction with the water tracker, instead of first launch.
I'd move this to a separated file:
```swift
enum UserDefaultsConstant {
    static let firstUserInteraction = "firstUserInteraction"
}
```